### ISSUE TITLE
Fractional Brownian Motion

### ISSUE THREAD

**Issue by oviedorodolfo** [SciML/DifferentialEquations.jl#779]
Support for fractional Brownian motion would be easy to add and very useful in some fields. I can only comment on financial derivatives valuation. Rough volatility models are a hot topic and they involve fractional Brownian motions. No closed-form solutions exist. Monte Carlo simulation is the only method available. Because it is super slow, these models cannot be used in real-time by financial institutions. They are beginning to simulate the output of the models for a dense grid of combinations of parameters and then fit a neural network model. The latter can be used in real-time by financial institutions to calibrate the model to the quotes of liquid financial instruments so as to infer the parameters of the model. Then the model is used to value non-liquid exotic financial instruments. What makes rough volatility so appealing is that it fits liquid short-term and long-term derivatives with few parameters and that the implied stochastic process of the underlying asset price also fits its historical statistical properties. Those are the only models that have all those properties. Modeling of derivatives on commodities, bonds, equities, indexes, interest rates, and many indexes could adopt these models. A site devoted to rough volatility: https://sites.google.com/site/roughvol/home. The pioneers of the new models, Jim Gatheral, presidential professor of finance at Baruch College, City University of New York, and Mathieu Rosenbaum, professor of quantitative finance at the École Polytechnique in Paris, earned the Quants of the Year Award 2021. By now only hedge funds have implemented these models. Banks with legacy models have not done so yet because of the costs of a new IT deployment. It would almost certainly be worthwhile for newcomers.

**Comment by ChrisRackauckas** [comment]
Yes, this is a good idea and definitely something we should add. I think the first place to support it would be NeuralPDE.jl
